src/games/brain-gcd.js

```diff
@@ -0,0 +1,29 @@
+import getRandomNumber from '../randomCount.js';
+import gamesCore from '../index.js';
+
+const gameDescription = 'Find the greatest common divisor of given numbers.';
+
+const getMaximumDivisior = (one, two) => {
```

Comment:
The name of your function tells me that a number will be returned, but a string is returned. In this case, it is better to convert to a string in the generator.

Reply:
See how recursion can make it easier to get a result
```
const getMaximumDivisior = (num1, num2) => {
  if (num2 === 0) {
    return num1;
  }
  return getGcd(num2, num1 % num2);
};
```

Reply:
ok, fixed!